starlette/testclient.py

```diff
@@ -37,16 +38,29 @@
         "You can install this with:\n"
         "    $ pip install httpx\n"
     )
+
+
+Auth: TypeAlias = "httpx.Auth | tuple[str | bytes, str | bytes]"
+QueryParams: TypeAlias = "httpx.QueryParams | typing.Mapping[str, str] | None"
+Cookies: TypeAlias = "httpx.Cookies | CookieJar | dict[str, str] | list[tuple[str, str]] | None"
+URL: TypeAlias = "httpx.URL | str"
+Timeout: TypeAlias = "float | httpx.Timeout | tuple[float | None, float | None, float | None, float | None] | None"
+Headers = typing.Union[
+    httpx.Headers,
+    typing.Mapping[bytes, bytes],
+    typing.Mapping[str, str],
+    typing.Sequence[typing.Tuple[bytes, bytes]],
+    typing.Sequence[typing.Tuple[str, str]],
+    None,
+]
```

Comment:
I pretty much have to do what httpx is doing on the `_client` module because I think we want to support the type possibilities that are there...